src/Analyzers/VisualBasic/CodeFixes/UseObjectInitializer/UseInitializerHelpers.vb

```diff
@@ -21,5 +22,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UseObjectInitializer
                                   WithInitializer(initializer).
                                   WithTrailingTrivia(objectCreation.GetTrailingTrivia())
         End Function
+
+        Public Sub AddExistingItems(objectCreation As ObjectCreationExpressionSyntax, nodesAndTokens As ArrayBuilder(Of SyntaxNodeOrToken))
+            If TypeOf objectCreation.Initializer Is ObjectMemberInitializerSyntax Then
+                Dim memberInitializer = DirectCast(objectCreation.Initializer, ObjectMemberInitializerSyntax)
+                nodesAndTokens.AddRange(memberInitializer.Initializers.GetWithSeparators())
+            ElseIf TypeOf objectCreation.Initializer Is ObjectCOllectioninitializersyntax Then
+                Dim collectionInitializer = DirectCast(objectCreation.Initializer, ObjectCollectionInitializerSyntax)
+                nodesAndTokens.AddRange(collectionInitializer.Initializer.Initializers.GetWithSeparators())
+            End If
+
+            If nodesAndTokens.Count Mod 2 = 1 Then
```

Comment:
```suggestion
            ' If we have an odd number of elements already, add a comma at the end so that we can add the rest of the
            ' items afterwards without a syntax issue.
            If nodesAndTokens.Count Mod 2 = 1 Then
```